code-challenges.test.js

```diff
@@ -34,8 +65,27 @@ const letterA = "a"
 const letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
 
-// b) Create the function that makes the test pass.
+describe("fruitChecker", () => {
+    it ("takes in an array of words and a single letter and returns an array of all the words containing that particular letter", () => {
+      expect(fruitChecker(letterA)).toEqual(["Mango", "Apricot", "Peach"]) 
+      expect(fruitChecker(letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+    })
+  })
+
+//   FAIL  ./code-challenges.test.js
+//   fruitChecker
+//     ✕ takes in an array of words and a single letter and returns an array of all the words containing that particular letter
+
+//   ● fruitChecker › takes in an array of words and a single letter and returns an array of all the words containing that particular letter
 
+//     ReferenceError: fruitChecker is not defined
+
+// b) Create the function that makes the test pass.
+const fruitChecker = (letter) => {
+    const letterFilter = fruitArray.filter(letter) 
+}
+console.log(fruitChecker(letterA))
+console.log(fruitChecker(letterE))
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind.
 
```

Comment:
The prompt gives us what a Full House is. So conditionally I would want to iterate over the array given and determine whether it is a full house and return true or false